tools/interop_matrix/client_matrix.py

```diff
@@ -281,8 +281,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
         ('v1.16.0', ReleaseInfo(testcases_file='php__v1.0.1')),
         ('v1.17.1', ReleaseInfo(testcases_file='php__v1.0.1')),
         ('v1.18.0', ReleaseInfo()),
-        # TODO:https://github.com/grpc/grpc/issues/18264
```

Comment:
Please leave a comment why v1.19 an v1.20 are missing so it's obvious this is not a mistake.

Reply:
Done.